gourmet/reccard.py

```diff
@@ -1544,6 +1544,7 @@ def set_from_fileCB (self, *args):
         debug("set_from_fileCB (self, *args):",5)
         f=de.select_image("Select Image",action=Gtk.FileChooserAction.OPEN)
         if f:
+            f, = f
```

Comment:
The same goes for here. This breaks if there are multiple values. This might be prevented somewhere else but let's plan for that failure here. I would be in favor of comment + assertion, but I guess either is fine.